### ISSUE TITLE
[Feature request] Could you create a button for background service reset  cleary?

### ISSUE THREAD

**Issue by redburn82** [bk138/droidVNC-NG#51]
This request from a last comment of #46: pull request.
Please consider to make a button for about clean restart of background service.

-----
Thank you for that you consider this ~~un-common~~ issue. 
( Frankly I didn't notice "Inputs not working after some time #38" ever. :-o )

Actually i already knew your code because it was one of the 2 solutions i though about.
It would be working fine but i have a worry something.
Because i've encountered another patten of exception but not caught it exactly so.. 
Anyway the easy way for handling or escaping from un-handled exception might be needed i think.

For example, if droidVNC has a button explicitly for clean restart background service include every states or variables, it could be more stable when unknown exceptions occures.

Could you consider about it?

Thanks.

_Originally posted by @redburn82 in https://github.com/bk138/droidVNC-NG/issues/46#issuecomment-950459147_

**Comment by bk138** [comment]
I would rather have some form of automatic recovery - in which specific situations should the restart be necessary?

**Comment by redburn82** [comment]
Actually, i've been using "droidVNC" as  sub-infortainment system on tesla browser.
The most important thing of this case, i have to use vnc interface when i'm on driving. 
So if a unexpected error has occured, i need a method for making it normal immediately.

But when i encountered "Inputs not working after some time" issue before analysing this issue,
automatic recovery has seem not to be working well.
I had to do "App Data & Cache clear" because it was a only way for resolve at that time.
I could't make it normal with some generic way as i can do as user. like app force close or so on. 

I did **"hamburger menu >> close all" on my android phone, but droidvnc background service was still alived**. 
it mean that if there is a problem on background service not control pannel then **user can't resolve it easy**.

Plus, **user can't know a timing when it be fixed after unexpected error has occured**.
so i think auto-recover is not enough  when droidVNC has to be turned on always.
for example using GPS navigator with droidVNC.
